README.md

```diff
@@ -64,11 +64,13 @@ to ``ovirt_engine_setup_answer_file_path`` variable.
 | ovirt_engine_setup_package_list       | []                    | List of extra packages to be installed on engine apart from `ovirt-engine` package. |
 | ovirt_engine_setup_fqdn               | UNDEF                 | Host fully qualified DNS name of the server. |
 | ovirt_engine_setup_organization       | UNDEF                 | Organization name for certificate. |
+| ovirt_engine_overwrite_pki_files      | False                 | Whether to overwrite existing PKI files from previous engine deployment. Setting this to `False` will make the installation abort if engine was previously deployed on target machone. |
 | ovirt_engine_setup_firewall_manager   | firewalld             | Specify the type of firewall manager to configure on Engine host, following values are available: `firewalld`,`iptables` or empty value (`null`) to skip firewall configuration. |
 | ovirt_engine_setup_require_rollback   | UNDEF                 | If `True`, setup will require to be able to rollback new packages in case of a failure. If not specified, the default answer from `engine-setup` will be used. Valid for updating/upgrading. |
 | ovirt_engine_setup_admin_password     | UNDEF                 | Password for the automatically created administrative user of the oVirt Engine.
 | ovirt_engine_setup_wait_running_tasks | False                 | If `True`, engine-setup will wait for running tasks to finish. Valid for `ovirt_engine_setup_version` >= 4.2. |
 | ovirt_engine_cinderlib_enable         | False                 | If `True`, cinderlib is enabled. Valid for `ovirt_engine_setup_version` >= 4.3. |
+| ovirt_engine_grafana_enable           | True                  | If `True`, Grafana integration will be set up. Valid for `ovirt_engine_setup_version` >= 4.4. |
```

Comment:
Grafana is mandatory component of oVirt 4.4, do we have the same question in engine-setup?

Anyway it would be better to add all related Grafana settings from engine-setup to this role using single RFE ...

Reply:
Hi @mwperina, is that really the case? During `engine-setup`, there is a question whether you want to configure Grafana integration or not. Admittedly, the default answer is Yes. Nonetheless I did several deployments of oVirt 4.4 without Grafana and they seemed to be running just fine. 

As far as I know, there aren't other Grafana-related options in `engine-setup`. However, if you prefer to handle that via more structured RFE, that's fine. I'll just keep these changes in my own branch and use it where I need it.

Reply:
> Hi @mwperina, is that really the case? During engine-setup, there is a question whether you want to configure Grafana integration or not. Admittedly, the default answer is Yes. Nonetheless I did several deployments of oVirt 4.4 without Grafana and they seemed to be running just fine.

Yes, but AFAIK if you decided to not configure Grafana/DWH on engine withing engine-setup, then you should configure it on a remote hosts. Sandro, am I right?

> As far as I know, there aren't other Grafana-related options in engine-setup. However, if you prefer to handle that via more structured RFE, that's fine. I'll just keep these changes in my own branch and use it where I need it.

No problem to merge it partially, but let's not forget to finish it properly. Martin, could you please create issue to add all relevant Grafana options to the role?

@sandrobonazzola @mnecas